source/Mole/UndirectedEdge.class.st

```diff
@@ -43,23 +70,31 @@ UndirectedEdge >> isDirected [
 { #category : #testing }
 UndirectedEdge >> isIncidentTo: aVertex [
 
-	^ relatedVertices includes: aVertex
+	^ endpoints includes: aVertex
+]
+
+{ #category : #private }
+UndirectedEdge >> isLabeledLike: anEdge [
+
+	^ self
```

Comment:
I you have access to the optional you can do something like:
`^ (labelOptional with: anEdge labelOptional whenBothUsedReturn: [:label :anotherLabel | label = anotherLabel])
          withContentDo: [:aBoolean |  aBoolean ] ifUnused: [true ]`

Also the implementation is equal in DirectedEdge.